### ISSUE TITLE
Adding a testing framework

### ISSUE THREAD

**Issue by ameech** [dolfelt/scheduler-js#3]
Any preferences with testing frameworks? I've had good experiences with Jasmine and Mocha/Chair.

I also like using https://github.com/airportyh/testem, it allows us to test in multiple browsers at the same time as we are coding.

Doing a bit more research I see Jasmine has [ajax support](http://jasmine.github.io/2.3/ajax.html), not sure if Mocha does, but I assume it does.

@dolfelt @shadowhand @curtis1000
